### ISSUE TITLE
goldchainc panics if there is no daemon running

### ISSUE THREAD

**Issue by LeeSmet** [nbh-digital/goldchain#61]
```
panic: failed to load constants from daemon's server and explorer modules: no response from daemon

goroutine 1 [running]:
github.com/nbh-digital/goldchain/vendor/github.com/threefoldtech/rivine/extensions/minting/client.CreateExploreCmd(0xc0000b8230)
        /home/lee/go/src/github.com/nbh-digital/goldchain/vendor/github.com/threefoldtech/rivine/extensions/minting/client/query.go:19 +0x1d7
main.main()
        /home/lee/go/src/github.com/nbh-digital/goldchain/cmd/goldchainc/main.go:29 +0x1c9
```

It would be better if we just show the error message and exit normally

**Comment by GlenDC** [comment]
Is a Rivine issue though, nothing to do directly with goldchain.

**Comment by GlenDC** [comment]
Resolved by f7e4923052bac5f95167d061db87468f6a62ae08.
